source/app/service-providers/fsal/util/file-parser.ts

```diff
@@ -82,8 +82,9 @@ export default function getMarkdownFileParser (
       file.firstHeading = firstH1.content
     }
 
-    file.wordCount = countWords(ast)
-    file.charCount = countChars(ast)
+    const locale = undefined
```

Comment:
Pro-tip: The AppServiceContainer singleton can now be retrieved from anywhere in the app, meaning you can retrieve the correct locale here, too!